src/components/definition-list/dd.d.ts

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { Dd } from '../definition-list';
```

Comment:
This doesn't need to be imported here, I'm surprised eslint isn't complaining about an unused import!

Reply:
✅